test/integration/scheduler/filters/filters_test.go

```diff
@@ -1758,6 +1759,111 @@ func TestUnschedulablePodBecomesSchedulable(t *testing.T) {
 			},
 			enableReadWriteOncePod: true,
 		},
+		{
+			name: "pod with pvc has node-affinity to non-existent/illegal nodes",
+			init: func(cs kubernetes.Interface, ns string) error {
+				storage := v1.ResourceRequirements{Requests: v1.ResourceList{v1.ResourceStorage: resource.MustParse("1Mi")}}
+				volType := v1.HostPathDirectoryOrCreate
+				pv, err := testutils.CreatePV(cs, st.MakePersistentVolume().
+					Name("pv-has-non-existent-nodes").
+					AccessModes([]v1.PersistentVolumeAccessMode{v1.ReadWriteMany}).
```

Comment:
I guess the access mode doesn't matter for the purpose of this test?

Reply:
hi @alculquicondor , 
Using ReadWriteOncePod access mode in 1.28 and 1.27 will report the following error:
```
cannot create pv: PersistentVolume "pv-has-non-existent-nodes" is invalid: spec.accessModes: Unsupported value: "ReadWriteOncePod": supported values: "ReadOnlyMany", "ReadWriteMany", "ReadWriteOnce"
```

`ReadWriteOncePod` is fearue gate in 1.28,1.27, so i change this to ReadWriteMany
https://github.com/kubernetes/kubernetes/blob/73dadc3b15efe6074b03b3f322bc33809abe31da/pkg/apis/core/validation/validation.go#L1670-L1673